adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -96,7 +95,7 @@ void HybrisProximityAdaptor::sendInitialData()
                ProximityData *d = buffer->nextSlot();
                d->timestamp_ = Utils::getTimeStamp();
                d->withinProximity_ = near;
-               d->value_ = near ? 10 : 0;
+               d->value_ = absinfo.value;
```

Comment:
Also, is it intentional that we assume that the sensor is covered if the initial probing fails? If the change event processing
is failing too it would lead to permanent covered state which would mean display will not turn on under various scenarios.

And why the fd is closed after the probe? How are the change events received? If it is via fd that is opened later, the initial probing done here might not be up to date anymore.